### DISCUSSION TITLE
Trace model without softmax

### DISCUSSION THREAD

**Discussion by olegsij** [open-mmlab/mmdeploy#1998] [Q&A]
Is there any way to trace mmseg model without last layer? It worked this way before mmseg 1.0 release, but in new version code for script converting mmseg model to torchscript was not updated. I found such functionality in this repo, but did not find a way to convert model without softmax
Something like this https://github.com/open-mmlab/mmdeploy/pull/1379/files/cff76c50f50ef0ec8e84a0df18398ed9c5f62199
but for 1.0.0 version

**Comment by AllentDan** [comment]
Hi, this is neglected by MMDeploy. Maybe you can pull a merge request to MMDeploy to fix it in the main branch?
